### PR TITLE
fix #71665 vanityfair.com

### LINKED ISSUE #71665: vanityfair.com
### Issue URL (Annoyance)
 [https://www.vanityfair.com/hollywood/2015/12/star-wars-force-awakens-review](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.vanityfair.com%2Fhollywood%2F2015%2F12%2Fstar-wars-force-awakens-review)
### Comment
Username: @Rakleed
> Email newsletter 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.4
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by stanislav-atr**
#71665

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1674,6 +1674,7 @@ telegraf.com.ua##.article__social_block
 20minutes.fr###header_banner_serious
 coincodex.com##app-sidebar-subscribe
 minfin.com.ua##.mf-subscribe--wgt
+vanityfair.com##.newsletter-subscribe-form
```
Comment (zloyden) [inline AnnoyancesFilter/sections/annoyances.txt:1677]: We have similar rules.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -109,6 +109,7 @@ jeotermalhaberler.com#$##subscribeModal { display: none!important; }
 jeotermalhaberler.com#$#.modal-backdrop { display: none!important; }
 jeotermalhaberler.com#$#body { overflow: visible!important; }
 glavkniga.ru##.popup_banner_area
+vanityfair.com##.persistent-bottom
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:112]: I don't think that we should block it. @AdguardTeam/filters-maintainers any thoughts on that?

Comment (AdamWr) [reply AnnoyancesFilter/sections/popups.txt:112]: There is also a "Subscribe" button on the top, so I think that we can block it.
It's annoying in my opinion.

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt:112]: I agree that is really annoying. Ok, let's block it, if others don't mind.

**Review by zloyden — APPROVED** [review]

**Pull request merged**